src/record/record.js

```diff
@@ -234,6 +255,9 @@ Record.prototype.unsubscribe = function( pathOrCallback, callback ) {
  * @returns {void}
  */
 Record.prototype.discard = function() {
+	if( this._checkDestroyed( 'discard' ) ) {
```

Comment:
Just following the convention used for the rest of the code... not sure how you would like to have it?